crates/db_views/site/src/impls.rs

```diff
@@ -40,7 +40,7 @@ impl SiteView {
         Ok(local_site)
       })
       .await
-      .map_err(|_e: Arc<LemmyError>| LemmyErrorType::LocalSiteNotSetup.into())
+      .map_err(|e| anyhow::anyhow!("err getting local site: {e:?}"))
```

Comment:
This means the error will always get wrapped in `anyhow::Error` which means it always gets `LemmyErrorType::Unknown`. But there is a method `Site::read_local` which is uncached and would work. In fact we can remove that method, replace callers with `SiteView::read_local` and write the query directly in `setup_local_site()` so that other code doesnt use the uncached method.